ynput/ayon-launcher
# Missing dependency

## Changelog Description
We're missing `colorsys` in output build which is requirement for OpenTimelineIO.

## Additional info
Module `colorsys` is built-in module, and there is no way how to tell python to include all built-in modules, because of that I've added all needed built-in modules to the list (to be sure).

## Testing notes:
1. Build and make installer.
2. Upload to server.
3. Module `colorsys` should be possible to import. (And OpenTimelineIO should work)
